linters/.jshintrc

```diff
@@ -1,26 +1,13 @@
 {
-  "globals": {
-    "__dirname": false,
-    "__filename": false,
-    "console": false,
-    "module": false,
-    "process": false,
-    "require": false
-  },
-  "camelcase": true,
   "bitwise": true,
   "browser": true,
   "curly": true,
   "eqeqeq": true,
   "freeze": true,
-  "immed": true,
-  "indent": 2,
   "latedef": true,
-  "newcap": true,
-  "noempty": true,
-  "quotmark": true,
-  "trailing": true,
+  "node": true,
+  "notypeof": true,
```

Comment:
That's actually not what it does, but we should actually remove this! 

I didn't realize this *suppresses* warnings (not sure why this isn't under relaxing options, but whatever...).